### ISSUE TITLE
yarn 2 workspace monorepo: `private_source_authentication_failure` error on registry.npmjs.org

### ISSUE THREAD

**Issue by jtbandes** [dependabot/dependabot-core#3702]
In the [foxglove/studio](https://github.com/foxglove/studio) repo, we are receiving some updates from dependabot, but the number of [PRs opened by dependabot](https://github.com/foxglove/studio/pulls?q=is%3Apr+author%3Aapp%2Fdependabot+) is lower than expected and definitely not limited by our `open-pull-requests-limit: 100`.

There are unexplained errors that show up in the log:
<img width="898" alt="image" src="https://user-images.githubusercontent.com/14237/117887952-2ea93880-b24d-11eb-8dfd-387de3c1db26.png">

Full log output is here: https://gist.github.com/jtbandes/766fdc3396c5e9036577fb5d4881ef17

Example error that may be related to internal packages (note `@foxglove/electron-socket`, below, is a package [in our monorepo](https://github.com/foxglove/studio/tree/main/packages/electron-socket))

```
...
updater | INFO <job_131018418> Updating @pmmmwh/react-refresh-webpack-plugin from 0.5.0-beta.1 to 0.5.0-beta.7
  proxy | 2021/05/11 06:14:49 [034] GET https://registry.yarnpkg.com:443/@actions%2fcore
  proxy | 2021/05/11 06:14:50 [034] 200 https://registry.yarnpkg.com:443/@actions%2fcore
...
  proxy | 2021/05/11 06:15:20 [084] GET https://registry.yarnpkg.com:443/@babel%2fplugin-transform-typescript
  proxy | 2021/05/11 06:15:20 [084] 200 https://registry.yarnpkg.com:443/@babel%2fplugin-transform-typescript
  proxy | 2021/05/11 06:15:20 [086] GET https://registry.yarnpkg.com:443/@babel%2fplugin-syntax-typescript
  proxy | 2021/05/11 06:15:20 [086] 200 https://registry.yarnpkg.com:443/@babel%2fplugin-syntax-typescript
  proxy | 2021/05/11 06:15:20 [088] GET https://registry.yarnpkg.com:443/@foxglove%2felectron-socket
  proxy | 2021/05/11 06:15:20 [088] 404 https://registry.yarnpkg.com:443/@foxglove%2felectron-socket
updater | INFO <job_131018418> Handled error whilst updating @pmmmwh/react-refresh-webpack-plugin: private_source_authentication_failure {:source=>"registry.npmjs.org"}
...
```

References #1297 for yarn 2

**Package ecosystem**
npm
**Package manager version**
npm 7, yarn 2
**Language version**
node 14/15
**Manifest location and content prior to update**
<details>
<summary>Links to package.json/yarn.lock files in monorepo</summary>

https://github.com/foxglove/studio/blob/main/package.json
https://github.com/foxglove/studio/blob/main/yarn.lock
https://github.com/foxglove/studio/blob/main/app/package.json
https://github.com/foxglove/studio/blob/main/packages/just-fetch/package.json
https://github.com/foxglove/studio/blob/main/packages/log/package.json
https://github.com/foxglove/studio/blob/main/packages/ros1/package.json
https://github.com/foxglove/studio/blob/main/packages/rosmsg-deser/package.json
https://github.com/foxglove/studio/blob/main/packages/velodyne-cloud/package.json
https://github.com/foxglove/studio/blob/main/packages/wasm-bz2/package.json
https://github.com/foxglove/studio/blob/main/packages/electron-socket/package.json

</details>

**dependabot.yml content**
https://github.com/foxglove/studio/blob/main/.github/dependabot.yml

**Comment by asciimike** [comment]
As per #1297, we don't support yarn v2, so if yarn v2 is required to fix this, it's unlikely that this will get fixed before then. I haven't looked at it in depth since I rejected it immediately as "won't fix yarn 2", but given that it's an auth problem, I'm not sure if it has anything to do with auth. Looking at https://www.npmjs.com/package/@pmmmwh/react-refresh-webpack-plugin, it looks like it's public, so not sure why that's failing with an auth error?

**Comment by jeffwidman** [comment]
Closing, as it looks like [you moved off Dependabot for now](https://github.com/foxglove/studio/pull/1407).

Work on #1297 is underway, so once that lands if you're still having problems please file an issue. I'd say re-open this one, but I really doubt the info in this issue will be relevant anymore once that lands... even for errors the logs will be completely different.

**Comment by fjakobs** [comment]
I'm having the same problem as @jtbandes. 

Re-opening the issues since https://github.com/dependabot/dependabot-core/issues/1297 has been closed and the problem persists.

**Comment by deivid-rodriguez** [comment]
Hei @fjakobs!

As @jeffwidman explained, Yarn v2 support has landed recently and that's a significant change over our previous code base. So even if the problem still exists, error logs and other relevant information for us to troubleshoot is likely be different now.

So we'd rather work with a fresh bug report. Could you open a separate ticket following our current bug report template, including as much information as possible?

Thank you!
